src/configure/configurers/remoteConfigurer.ts

```diff
@@ -0,0 +1,190 @@
+            });
+            } else {
+                telemetryHelper.setTelemetry(TelemetryKeys.PipelineDiscarded, 'true');
+                throw new UserCancelledError(Messages.operationCancelled);
+            }
+
+        if ( provisioningServiceResponse != undefined) {
+            this.setQueuedPipelineUrl(provisioningServiceResponse, inputs);
+        } else {
+            throw new Error("Failed to configure provisoining pipeline");
+        }
+    }
+
+    public async showPipelineFiles(): Promise<void> {
+        this.filesToCommit.forEach(async (file) => {
+            await this.localGitRepoHelper.addContentToFile(file.content, file.absPath);
+            await vscode.window.showTextDocument(vscode.Uri.file(file.absPath));
+        });
+    }
+
+    public setQueuedPipelineUrl(provisioningConfiguration: ProvisioningConfiguration, inputs: WizardInputs){
+      const commitId = (provisioningConfiguration.result.pipelineConfiguration as CompletePipelineConfiguration).commitId;
+      this.queuedPipelineUrl = `https://github.com/${inputs.sourceRepository.repositoryId}/commit/${commitId}/checks`;
+    }
+
+    public async  getFileToCommit(draftPipelineConfiguration: DraftPipelineConfiguration): Promise<void> {
+        let destination: string;
+        for (const file of draftPipelineConfiguration.files ) {
+            destination = await this.getPathToFile(Path.basename(file.path), Path.dirname(file.path));
+            const decodedData = new Buffer(file.content, 'base64').toString('utf-8');
+            this.filesToCommit.push({absPath: destination, content: decodedData, path: file.path} as DraftFile);
+        }
+    }
+
+    public async getPathToFile( fileName: string, directory: string) {
+        const dirList = directory.split("/"); // Hardcoded as provisioning service is running on linux and we cannot use Path.sep as it is machine dependent
+        let directoryPath: string = "";
+        directoryPath = await this.localGitRepoHelper.getGitRootDirectory();
+        dirList.forEach((dir) => {
+            try {
+                directoryPath = Path.join(directoryPath, dir);
+                // tslint:disable-next-line:non-literal-fs-path
+                if (!fs.existsSync(directoryPath)) {
+                    // tslint:disable-next-line:non-literal-fs-path
+                    fs.mkdirSync(directoryPath);
+                }
+            }
+            catch (error) {
+                throw error;
+            }
+        });
+        telemetryHelper.setTelemetry(TelemetryKeys.WorkflowFileName, fileName);
+        return Path.join(directoryPath, fileName);
+    }
+
+    public async CreatePreRequisiteParams(wizardInputs: WizardInputs): Promise<void>{
+        const parsedCredentials: { [key: string]: any } = JSON.parse(JSON.stringify(wizardInputs.azureSession.credentials));
+        wizardInputs.pipelineConfiguration.params["armAuthToken"] = "Bearer " + parsedCredentials["tokenCache"]["target"]["_entries"][0]["accessToken"];
```

Comment:
Can you try parsing via https://github.com/Azure/azure-sdk-for-node/blob/master/runtime/ms-rest/lib/credentials/tokenCredentials.js

Actually credentials are of type https://github.com/Azure/azure-sdk-for-node/blob/master/runtime/ms-rest/lib/serviceClient.js#L124

Reply:
Yes, we tried with TokenCredentials but in its d.ts file, we have no way to access token param from this class. So last option left was parse it to [string]any then take value from it. We can have a chat/call to discuss this.